### ISSUE TITLE
No module named 'inplace_abn'

### ISSUE THREAD

**Issue by ZhaoNan1** [MichaelFan01/STDC-Seg#69]
训练的时候遇到这个问题，有什么办法解决吗？
python==3.5.6
pytorch==1.1

**Comment by zjsun7** [comment]
请问问题解决了吗，我也遇到同样的问题？

**Comment by zjsun7** [comment]
> 训练的时候遇到这个问题，有什么办法解决吗？ python==3.5.6 pytorch==1.1

老哥！

**Comment by chen1234520** [comment]
放弃inplace_abn ，将所有的inplace_abn替换为官方bn

**Comment by zjsun7** [comment]
完美解决，放弃inplace_abn ，将所有的inplace_abn替换为官方bn。

**Comment by JJY0710** [comment]
> 完美解决，放弃inplace_abn ，将所有的inplace_abn替换为官方bn。

请问具体是怎么更换的，能详细解释一下吗，没太懂functions.py调用的cpp

**Comment by reevesi** [comment]
fix all of the  comment in  model_stages.py , like
#import torch.nn.BatchNorm2d as BatchNorm2d
BatchNorm2d = nn.BatchNorm2d

**Comment by CiciliaLi** [comment]
> 放弃inplace_abn ，将所有的inplace_abn替换为官方bn

为啥可以直接替换成bn，inplace_abn不是bn+激活吗？

**Comment by jiaoluonanhai** [comment]
> 完美解决，放弃inplace_abn ，将所有的inplace_abn替换为官方bn。

inplace_abn_sync() 要换成什么？

**Comment by jiaoluonanhai** [comment]
> 完美解决，放弃inplace_abn ，将所有的inplace_abn替换为官方bn。

inplace_abn_sync() 要换成什么？

**Comment by XYZach** [comment]
> 完美解决，放弃inplace_abn ，将所有的inplace_abn替换为官方bn。

想请问一下原本的inplace_abn的区别或者意义是什么知道吗？
